src/PhpBrew/Downloader/UrlDownloader.php

```diff
@@ -53,12 +54,14 @@ public function download($url, $targetFilePath)
 
             // check for wget or curl for downloading the php source archive
             // TODO: use findbin
-            if (exec('command -v wget')) {
-                if(system('wget --no-check-certificate -c -O ' . $targetFilePath . ' ' . $url) !== false){
```

Comment:
`system('wget --no-check-certificate -c -O ' . $targetFilePath . ' ' . $url) !== false` must be inverted because `system` function returns `false` only when it fails.